src/index.html

```diff
@@ -8,6 +8,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+
+      <img src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)">
+
+      <h2 class="card__info">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__reviws">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviws-count">Reviews: 5</div>
+
+      </div>
+
+      <div class="card__price-conteiner">
+        <div class="card__price-lable">Price:</div>
+        <div class="card__price">$2,199</div>
```

Comment:
use semantic tags for text

Reply:
<img width="651" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/a4c1293c-2d0f-4798-b6fe-5bffd4a04d9e">